index.bs

```diff
@@ -837,11 +837,26 @@ When an {{XRReferenceSpace}} is requested, the user agent MUST <dfn>create a ref
 
   1. Let |session| be the {{XRSession}} object that requested creation of a reference space.
   1. Let |type| be set to the {{XRReferenceSpaceType}} passed to {{requestReferenceSpace()}}.
-  1. If |type| is {{bounded-floor}}, let |referenceSpace| be a new {{XRBoundedReferenceSpace}}.
-  1. Else let |referenceSpace| be a new {{XRReferenceSpace}}.
-  1. Initialize |referenceSpace|'s [=XRReferenceSpace/type=] to be |type|.
-  1. Initialize [=XRSpace/session=] to be |session|.
-  1. Return |referenceSpace|.
+  1. If the [=reference space is supported=] for |type| and |session|, run the following steps:
+    1. If |type| is {{bounded-floor}}, let |referenceSpace| be a new {{XRBoundedReferenceSpace}}.
+    1. Else let |referenceSpace| be a new {{XRReferenceSpace}}.
+    1. Initialize |referenceSpace|'s [=XRReferenceSpace/type=] to be |type|.
+    1. Initialize [=XRSpace/session=] to be |session|.
```

Comment:
Should say `Initialize |referenceSpace|'s [=XRSpace/session=] to be |session|.`

Reply:
This doesn't look like it was actually resolved in the latest commit?

Reply:
no idea what happened there, but fixed